CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - PR #2098 Align DataFrame and Series indices before executing binary ops
 
 ## Improvements
+- PR #2109 added name support for string column
```

Comment:
Fix the formatting here, should be one empty line below Improvements and then no empty lines in between entries